### PR TITLE
React Calculator Project - Milestone II

### PR DESCRIPTION
**Pull request by necmigunduz**
In this project, I did:
- Add Button, ButtonPanel, and Display components to the project, 
- Fix babel-loader problem,
- Fix linters.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @necmigunduz  👋

**You've done pretty well here. Using the right linter config with Button and Display components well-written. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Display />
+      <ButtonPanel />
+    </div>
```
Comment (mikenath223) [inline src/components/App.js:11]: -  [ ] The Display and ButtonPanel components should be wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)).

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export default function Button(props) {
+  const { name, clickHandle } = props;
+  const handleClick = (button) => clickHandle(button);
+  return (
+    <div className="button">
+      <button
+        className="name"
+        type="button"
+        onClick={handleClick.bind(this, name)}
```
Comment (mikenath223) [inline src/components/Button.js:12]: - [ ] Since this is not a class component kindly remove the `.bind` statement.

**Comment by mikenath223** [comment]
- [ ] Also, please remove the `src/App.js` component since there is an `App.js` in the `src/components` directory.

**Review by Berabjesus — APPROVED** [review]
# **Status: PROJECT APPROVED** 
Hi @necmigunduz 👋 congratulations on completing this milestone.
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete. 👍** 

I wish you all the very best in your next tasks:1st_place_medal:
#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Pull request merged**